mypy/test/teststubgen.py

```diff
@@ -64,6 +64,23 @@ def test_packages_found(self) -> None:
             finally:
                 os.chdir(current)
 
+    def test_module_not_found(self) -> None:
+        current = os.getcwd()
+        with tempfile.TemporaryDirectory() as tmp:
+            try:
+                import io
+                capturedOutput = io.StringIO()
```

Comment:
This is a non-typical case for Python variables, I would propose to use `captured_output`.